tests/rest/client/sliding_sync/test_extensions.py

```diff
@@ -120,19 +120,19 @@ def test_extensions_lists_rooms_relevant_rooms(
                 "foo-list": {
                     "ranges": [[0, 1]],
                     "required_state": [],
-                    "timeline_limit": 0,
+                    "timeline_limit": 1,
```

Comment:
I assume this is why you changed it:

(no wrapped properly)

```suggestion
                    # We set this to `1` because we're testing `receipts` which interact
                    # with the `timeline`. With receipts, when a room hasn't been sent
                    # down the connection before or it appears as `initial: true`, we
                    # only include receipts for events in the timeline to avoid bloating
                    # and blowing up the sync response as the number of users in the
                    # room increases. (this behavior is part of the spec)
                    "timeline_limit": 1,
```